uswds/uswds-site
# USWDS-Site - Research page: Edit testing day checklist copy

# Summary

Incorporated small edits for the testing day item description on the research page, according to the edits in [this doc](https://docs.google.com/document/d/1Lx-oUHboZ33aUt4XYyNU--SB_1_6Je-LHyLYHVRRCD0/edit) (Google docs :lock:).

## Preview link

Preview link: [Research page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-research-page-copyedit/about/research/#key-research-and-research-operations)

## Testing and review

- Confirm that the update matches the update in the document
- Confirm that the content makes sense and is free from spelling and grammatical error
